#--- siege-2.61/doc/siegerc.in~	2004-11-19 17:06:18.000000000 +0200
#+++ siege-2.61/doc/siegerc.in	2005-04-22 14:42:18.000000000 +0300
#@@ -14,6 +14,7 @@
# # You can also reference ENVIRONMENT variables without
# # actually declaring them, example:
# # logfile = $(HOME)/var/siege.log
#+logfile = /var/log/siege.log
# 
# #
# # Signify verbose mode, true turns on verbose output
--- ./doc/siegerc.in~	2005-08-17 14:06:51.000000000 +0300
+++ ./doc/siegerc.in	2005-08-17 14:25:03.000000000 +0300
@@ -59,7 +59,7 @@
 # Default logging status, true turns logging on.
 # ex: logging = true|false
 #
-logging = true
+logging = false
 
 #
 # Logfile, the default siege logfile is $PREFIX/var/siege.log
